src/redux/query_settings/sagas.js

```diff
@@ -228,6 +247,7 @@ export function* validateIncidentQueryImpl() {
     if (incidentStatus) params['statuses[]'] = incidentStatus;
     if (incidentUrgency) params['urgencies[]'] = incidentUrgency;
     if (teamIds.length) params['team_ids[]'] = teamIds;
+    if (escalationPolicyIds.length) params['escalation_policy_ids[]'] = escalationPolicyIds;
```

Comment:
I don't believe you can query by [EP ids](https://developer.pagerduty.com/api-reference/9d0b4b12e36f9-list-incidents), so this is redundant. 

We may have to update the query confirmation modal that it is an estimated number of results as we can't query this via the API (for another PR at another time!)

Reply:
Agree, that's redundant. I added `filterIncidentsByEscalationPolicyImpl` in `getIncidents()` similar to the Priority filter so it gets filtered after the initial GET.

```suggestion
```